mtprotoproxy.py

```diff
@@ -170,17 +212,17 @@ def init_config():
             modes["secure"] = False
 
     if not modes["classic"] and not modes["secure"] and not modes["tls"]:
-        print_err("No known modes enabled, enabling tls-only mode")
+        logger.error("No known modes enabled, enabling tls-only mode")
         modes["tls"] = True
 
     if legacy_warning:
-        print_err("Legacy options SECURE_ONLY or TLS_ONLY detected")
-        print_err("Please use MODES in your config instead:")
-        print_err("MODES = {")
-        print_err('    "classic": %s,' % modes["classic"])
-        print_err('    "secure": %s,' % modes["secure"])
-        print_err('    "tls": %s' % modes["tls"])
-        print_err("}")
+        logger.error("Legacy options SECURE_ONLY or TLS_ONLY detected")
+        logger.error("Please use MODES in your config instead:")
+        logger.error("MODES = {")
+        logger.error('    "classic": %s,', modes["classic"])
+        logger.error('    "secure": %s,', modes["secure"])
+        logger.error('    "tls": %s', modes["tls"])
+        logger.error("}")
```

Comment:
I'd propose to use single call to `logger.error` with multiline string literal:

```
logger.error("""Legacy options SECURE_ONLY or TLS_ONLY detected
Please use MODES in your config instead:
...
""", modes["classic"], modes["secure"], modes["tls"])
```
This way it will be semantically a single log message which is important for some logger backends like JSON etc